test/lib/formatter/RuleResultRecombinator.test.ts

```diff
@@ -107,28 +111,28 @@ const allFakeRuleResults: RuleResult[] = [
 			"endLine": 2,
 			"endColumn": 57,
 			"severity": 4,
-			"ruleName": "UnusedImports",
+			"ruleName": "ApexAssertionsShouldIncludeMessage",
```

Comment:
Changed these, because they must exist in the catalog for the sarif tests to pass.